src/ScottPlot4/ScottPlot/Plottable/Image.cs

```diff
@@ -63,45 +90,71 @@ public void ValidateData(bool deep = false)
             if (double.IsNaN(Y) || double.IsInfinity(Y))
                 throw new InvalidOperationException("y must be a real value");
 
+            if (SizeInAxisUnits is SizeF axisSize)
+            {
+                if (double.IsNaN(axisSize.Width) || double.IsInfinity(axisSize.Width))
```

Comment:
`double.IsNaN(foo) || double.IsInfinity(foo)` can be replaced by `!double.IsFinite(foo)`

Reply:
I am recalling that we cannot use `double.IsFinite` on .NET Framework, which ScottPlot supports.

Reply:
> we cannot use `double.IsFinite` on .NET Framework, which ScottPlot supports

Hopefully ScottPlot won't have to support .NET Framework forever 😜